File: LabSheet_11.1.c
// C Program to find the sum of digits of a number

#include <stdio.h>
int main()
{
    int num, sum = 0, digit;
    printf("Enter a number : ");
    scanf("%d", &num);
    while (num > 0)
    {
        digit = num % 10;
        sum = sum + digit;
        num = num / 10;
    }
    printf("Sum of the digits is %d", sum);
}

File: Semester 2 - Programming in Python/Labsheet 11/labsheet_11.1.py
# Python program to define an user-defined function and call that function

def addNumbers(a, b):
    return a + b

number1 = int(input("Enter first number: "))
number2 = int(input("Enter second number: "))

print(f"Sum of {number1} and {number2} is {addNumbers(number1, number2)}")